packages/lesswrong/components/common/ContentItemTruncated.tsx

```diff
@@ -31,11 +40,13 @@ const ContentItemTruncated = ({maxLengthWords, graceWords=20, expanded=false, ra
     } : truncateWithGrace(html, maxLengthWords, graceWords, rawWordCount);
   
   return <>
-    <ContentItemBody
-      dangerouslySetInnerHTML={{__html: truncatedHtml}}
-      className={className}
-      description={description}
-    />
+    <div className={maxHeight ? classes.maxHeight : null}>
```

Comment:
Slightly better to put this in the `className` that's on `ContentItemBody`, to avoid creating a bunch of spurious divs.